openapi/schemas/processes-core/statusCode.yaml

```diff
@@ -3,6 +3,6 @@ nullable: false
 enum:
```

Comment:
This list is in conflict with openEO. Didn't we agree to use the openEO codes?

Reply:
I think it was decided that status codes were to stay the way they currently are to avoid breaking Core v2.0 that is about to be submitted for approval. Changing them would require another round of tests and validation, and delay yet again its release already overdue. 

If I recall correctly, there was an idea that jobs submitted given a certain OAP/openEO/etc. pattern (eg: using `Content-Schema` or whatever) would be allowed to use their own set of status codes. Basically, the client interacting with the server could negotiate a certain interface. I think it makes sense if OAP-style inputs/outputs or openEO-style parameters/returns are employed when creating the job, the status codes should align with them. This might be the most portable approach, since even aligning status codes between OAP/openEO would end up breaking for any other interface that could be added later. For example, CWL have their own codes as well, and a client that knows CWL better might prefer its status code names.

Maybe a conformance class should be added, saying something along the lines of "if the job is created with openEO parameters/returns, the server MAY employ openEO-style status codes" + list the mapping shown in https://github.com/opengeospatial/ogcapi-processes/issues/420? Using `/conformance` listing that class, a client would be able to identify which status codes are to be expected.

Reply:
I understood it completely different, but I couldn't attend the last meeting. I thought because you are going 2.0 anyway you'd switch. Hmm.. not sure this all is worth it we we don't try to resolve the conflicts.

Reply:
I suggest bringing it up in the next meeting to validate. This Part 4 document simply adds `create`. The rest are "whatever Core defines".